test-case/multiple-pause-resume.sh

```diff
@@ -56,7 +56,7 @@ rnd_range=$((rnd_max - rnd_min))
 [[ $rnd_range -le 0 ]] && dlogw "Error random range scope [ min:$rnd_min - max:$rnd_max ]" && exit 2
 
 tplg=${OPT_VAL['t']}
-func_pipeline_export "$tplg" "type:any"
+func_pipeline_export "$tplg" "type:any & ~smart_amp"
```

Comment:
if you are going to filter something, it should be the echo reference. That's a clear case of a useless test. pause on playback makes a lot more sense.